tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/view/viewmodel/EventStoryViewModel.kt

```diff
@@ -0,0 +1,73 @@
+package band.effective.office.tv.view.viewmodel
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import band.effective.office.tv.domain.LatestEventInfoUiState
+import band.effective.office.tv.domain.models.Employee.Anniversary
+import band.effective.office.tv.domain.models.Employee.Birthday
+import band.effective.office.tv.domain.models.Employee.EmployeeInfo
+import band.effective.office.tv.domain.models.Employee.NewEmployee
+import band.effective.office.tv.network.use_cases.EmployeeInfoUseCase
+import band.effective.office.tv.utils.DateUtlils.getYearsWithTheCompany
+import band.effective.office.tv.utils.DateUtlils.isCelebrationToday
+import band.effective.office.tv.utils.DateUtlils.isNewEmployeeToday
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.*
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.catch
+import javax.inject.Inject
+
+@HiltViewModel
+class EventStoryViewModel @Inject constructor(private var employeeInfoUseCase: EmployeeInfoUseCase) :
+    ViewModel() {
+    val uiState =
```

Comment:
State should be private, see the example here
 https://github.com/effectiveband/Compose-Sample/blob/main/screens/feed/src/main/kotlin/band/effective/headlines/compose/feed/presentation/FeedViewModel.kt#L23